<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordEditor</class>
 <widget class="QDialog" name="PasswordEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pB_saveNewPassword">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>277</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="l_CurrentPassword">
        <property name="text">
         <string>Current Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_CurrentPassword"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="l_NewPassword">
        <property name="text">
         <string>New Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_NewPassword"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
